example_configs/stalwart.md

```diff
@@ -0,0 +1,44 @@
+# Stalwart Mailserver
+
+[Stalwart-mailserver](https://github.com/stalwartlabs/mail-server) is a Production-ready full-stack but simple mail server (SMTP, IMAP, LDAP, Antispam, Antivirus, etc.) written in Rust.
+
+To integrate with LLDAP, ensure you correctly add these ldap setting to your `config.toml`.
+
+## Config.toml File Sample - (only the ldap portion)
+```toml
+[storage]
+  directory = "ldap"
+
+[directory]
+  [directory.ldap]
+    base-dn = "dc=example,dc=org"
+    timeout = "30s"
+    type = "ldap"
+    url = "ldap://ldap.domain.example.org:3890"
+    [directory.ldap.attributes]
+      class = "objectClass"
+      email = "mail"
+      groups = "member"
+      name = "uid"
+      secret = "userPassword"
+      [directory.ldap.attributes.description]
+        0 = "displayName"
+    [directory.ldap.bind]
+      dn = "uid=admin,ou=people,dc=example,dc=org"
```

Comment:
What permission does the bind user require? Read only, change passwords, create/modify users?

Usually we recommend against binding directly as the admin, principle of least privilege.